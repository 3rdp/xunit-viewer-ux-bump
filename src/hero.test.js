import React from 'react'
import Hero from './hero'
import renderer from 'react-test-renderer'

test('renders hero', () => {
  const tree = renderer.create(<Hero />).toJSON()
  expect(tree).toMatchSnapshot()
})
