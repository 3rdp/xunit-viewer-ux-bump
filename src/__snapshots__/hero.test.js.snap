// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders hero 1`] = `
<section
  className="hero is-black"
>
  <div
    className="hero-body"
  >
    <div
      className="container hero-body-container"
    >
      <svg
        className="logo"
        fill="none"
        height="64"
        viewBox="0 0 64 64"
        width="64"
        xmlns="http://www.w3.org/2000/svg"
      >
        <rect
          fill="#2C3E50"
          height="64"
          width="64"
        />
        <path
          d="M19.0625 28.9688L24.5156 20.25H28.0469L20.8594 31.5312L28.2188 43H24.6562L19.0625 34.125L13.4375 43H9.89062L17.2656 31.5312L10.0625 20.25H13.5781L19.0625 28.9688Z"
          fill="#0DBF1F"
        />
        <path
          d="M42.4344 39.0156L48.9344 20.25H52.2156L43.7781 43H41.1219L32.7 20.25H35.9656L42.4344 39.0156Z"
          fill="#B32010"
        />
        <path
          clipRule="evenodd"
          d="M0 64L4 60V4H60L64 0H4H0V4V64Z"
          fill="#0DBF1F"
          fillRule="evenodd"
        />
        <path
          clipRule="evenodd"
          d="M0 64H60H64V60V0L60 4V60H4L0 64Z"
          fill="#B32010"
          fillRule="evenodd"
        />
      </svg>
      <h1
        className="title"
      >
        Xunit Viewer
      </h1>
    </div>
  </div>
</section>
`;
